.github/workflows/docs.yaml

```
@@ -183,12 +169,44 @@ jobs:
+            CLIENT_DIR=./docs/build/current/client
+          else
+            CLIENT_DIR=./docs/build/client
+          fi
+          mkdir -p "$CLIENT_DIR"
+          cp -r ./webapp/. "$CLIENT_DIR/"
 
-      - name: Deploy to gh-pages
-        uses: peaceiris/actions-gh-pages@v4
+      - name: Deploy to gh-pages (main branch)
+        if: github.event_name == 'push' && needs.check-repo.outputs.is-deploy-branch == 'true'
+        uses: peaceiris/actions-gh-pages@v3
```

actionlint (1.7.12)

[error] the runner of "peaceiris/actions-gh-pages@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)